src/powerbi-visual/webpack.config.dev.ts

```diff
@@ -1,228 +1,3 @@
-    libraryTarget: +powerbiApi.version.replace(/\./g, '') >= 320 ? 'var' : undefined
-  },
-  devServer: {
-    static: {
-      directory: path.join(__dirname, '.tmp', 'drop'), // path with assets for dev server, they are generated by webpack plugin
-      publicPath: '/assets'
-    },
-    compress: true,
-    port: 8080, // dev server port
-    hot: false,
-    https: {},
-    liveReload: false,
-    webSocketServer: false,
-    headers: {
-      'access-control-allow-origin': '*',
-      'cache-control': 'public, max-age=0'
-    }
-  },
-  externals:
-    powerbiApi.version.replace(/\./g, '') >= 320
-      ? {
-          'powerbi-visuals-api': 'null',
-          fakeDefine: 'false'
-        }
-      : {
-          'powerbi-visuals-api': 'null',
-          fakeDefine: 'false',
-          corePowerbiObject: "Function('return this.powerbi')()",
-          realWindow: "Function('return this')()"
-        },
-  plugins: [
-    new webpack.DefinePlugin({
-      __VUE_OPTIONS_API__: JSON.stringify(true),
-      __VUE_PROD_DEVTOOLS__: JSON.stringify(false)
-    }),
-    new VueLoaderPlugin(),
-    new MiniCssExtractPlugin({
-      filename: 'visual.css',
-      chunkFilename: '[id].css'
-    }),
-    new Visualizer({
-      reportFilename: statsLocation,
-      openAnalyzer: false,
-      analyzerMode: `static`
-    }),
-    // visual plugin regenerates with the visual source, but it does not require relaunching dev server
-    new webpack.WatchIgnorePlugin({
-      paths: [path.join(__dirname, pluginLocation), './.tmp/**/*.*']
-    }),
-    // custom visuals plugin instance with options
-    new PowerBICustomVisualsWebpackPlugin({
-      ...pbivizFile,
-      compression: 0,
-      capabilities: capabilitiesFile,
-      stringResources:
-        localizationFolders &&
-        localizationFolders.map((localization) =>
-          path.join(resourcesFolder, localization, 'resources.resjson')
-        ),
-      apiVersion: powerbiApi.version,
-      capabilitiesSchema: powerbiApi.schemas.capabilities,
-      pbivizSchema: powerbiApi.schemas.pbiviz,
-      stringResourcesSchema: powerbiApi.schemas.stringResources,
-      dependenciesSchema: powerbiApi.schemas.dependencies,
-      devMode: false,
-      generatePbiviz: false,
-      generateResources: true,
-      minifyJS: false,
-      minify: false,
-      modules: true,
-      visualSourceLocation: '../../src/visual',
-      pluginLocation: pluginLocation,
-      packageOutPath: path.join(__dirname, 'dist')
-    }),
-    new ExtraWatchWebpackPlugin({
-      files: [pbivizPath, capabilitiesPath]
-    }),
-    powerbiApi.version.replace(/\./g, '') >= 320
-      ? new webpack.ProvidePlugin({
-          define: 'fakeDefine'
-        })
-      : new webpack.ProvidePlugin({
-          window: 'realWindow',
-          define: 'fakeDefine',
-          powerbi: 'corePowerbiObject'
-        })
-  ]
-}
-
-export default config
+export default buildConfig({ mode: 'dev' })
```

Comment:
unnecessary that the config was duplicated twice when there's only like a few different lines of code